src/chocolatey/infrastructure.app/services/PowershellService.cs

```diff
@@ -302,9 +292,9 @@ private class PowerShellExecutionResults
             public bool StandardErrorWritten { get; set; }
         }
 
-        private PowerShellExecutionResults run_external_powershell(ChocolateyConfiguration configuration, string chocoPowerShellScript)
+        private PowershellService.PowerShellExecutionResults run_external_powershell(ChocolateyConfiguration configuration, string chocoPowerShellScript)

```

Comment:
I'm guessing a rebase failure. :-( ReSharper was muttering about it, but I thought that was how it was on master so I had left it there. Will fix.